testsuite/tests/kuadrant/authorino/operator/tls/test_webhook.py

```diff
@@ -0,0 +1,133 @@
+
+from testsuite.objects import Authorization
+from testsuite.certificates import CertInfo
+from testsuite.utils import cert_builder
+from testsuite.openshift.objects.validating_webhook import ValidatingWebhook
+from testsuite.openshift.objects.ingress import Ingress
+
+OPA_POLICY = """
+    request := json.unmarshal(input.context.request.http.body).request
+    verb := request.operation
+    ingress := request.object { verb == "CREATE" }
+    ingress := request.oldObject { verb == "DELETE" }
+    forbidden { count(object.get(ingress.spec, "rules", [])) == 0 }
+    rules { count(ingress.spec.rules) == 1; ingress.spec.rules[0] == {} }
+    allow { rules; not forbidden }
+"""
+
+
+@pytest.fixture(scope="session")
+def specific_authorino_name(blame):
+    """Define specific name for authorino which matches with name in authorino certificate"""
+    return blame('authorino')
+
+
+@pytest.fixture(scope="session")
+def authorino_domain(openshift, specific_authorino_name):
+    """
+    Hostname of the upstream certificate sent to be validated by APIcast
+    May be overwritten to configure different test cases
+    """
+    return f"{specific_authorino_name}-authorino-authorization.{openshift.project}.svc"
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, authorino_domain, wildcard_domain):
+    """Certificate hierarchy used for the tests.
+    Authorino certificate has *hosts* set to *authorino_domain* value.
+    """
+    chain = {
+        "envoy_ca": CertInfo(children={
+            "envoy_cert": None,
+            "valid_cert": None
+        }),
+        "authorino_ca": CertInfo(children={
+            "authorino_cert": CertInfo(hosts=authorino_domain),
+        }),
+        "invalid_ca": CertInfo(children={
+            "invalid_cert": None
+        })
+    }
+    return cert_builder(cfssl, chain, wildcard_domain)
+
+
+@pytest.fixture(scope="module")
+def authorino_parameters(authorino_parameters, specific_authorino_name):
+    """Setup TLS with specific name for authorino."""
+    authorino_parameters['name'] = specific_authorino_name
+    return authorino_parameters
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorization, openshift, module_label, authorino_domain) -> Authorization:
+    """In case of Authorino, AuthConfig used for authorization"""
+    authorization.remove_all_hosts()
+    authorization.add_host(authorino_domain)
+
+    authorization.identity.remove_all()
+    plain = {'authJSON': 'context.request.http.body.@fromstr|request.userInfo'}
+    authorization.identity.kubernetes('k8s-userinfo', plain)
+
+
+    authorization.authorization.opa_policy('features', OPA_POLICY)
+
+    when = [{'selector': 'auth.authorization.features.allow', 'operator': 'eq', 'value': 'true'},
+            {'selector': 'auth.authorization.features.verb', 'operator': 'eq', 'value': 'CREATE'}]
+    kube_user = {'valueFrom': {'authJSON': 'auth.identity.username'}}
```

Comment:
Similar here, you were right that there is an option to use constant value but there will most likely be a future refactor that will unify the behaviour of `valueFrom` and `value` and if we cut it down to just `auth.identity.username` I think it will be more readable and easier to work with